_includes/manuals/1.10/4.2.6_matchers.md

```diff
@@ -94,6 +88,9 @@ Note that the JSON hash syntax is not the same as Puppet's hash syntax: `{"examp
 
 #### Input validation
 
+The *Optional input validator* section can be used to restrict the allowed values for the parameter.  It is important to note that the validation applies to changes made from the Host edit page as well as the Puppet Classes edit page.
+The input validation section is hidden by default but can be opened by clicking on it's title. When changing the parameter type this section will be automatically expanded to chane the validations according to the new type.
```

Comment:
Add an extra line break to make this a new paragraph.  "it's" should be "its", "chane" => "change".